IMA/BasicIMAPlayer/objc/BasicIMAPlayer/BaseViewController.m

```diff
@@ -96,63 +92,24 @@ - (void)createPlayerView
     }
 }
 
-- (void)setup
+- (void)setupPlaybackController
 {
-    [self createPlayerView];
-
-    BCOVPlayerSDKManager *manager = [BCOVPlayerSDKManager sharedManager];
+    // NO-OP
```

Comment:
FYI, [this](https://developer.arm.com/documentation/dui0068/b/arm-instruction-reference/arm-pseudo-instructions/nop-arm-pseudo-instruction?lang=en) (ARM) and [this](https://www.felixcloutier.com/x86/nop) (Intel). 😁